composeApp/src/commonMain/kotlin/com/theseuntaylor/snippy/ui/components/SnippyButton.kt

```diff
@@ -16,6 +16,7 @@ import androidx.compose.ui.unit.dp
 fun SnippyButton(
     onClick: () -> Unit,
     textContent: String,
+    enabled: Boolean = true,
```

Comment:
Not sure how it is for y'all but buttons are usually enabled by default so we use a `disabled` prop instead for exceptions.